feature-libs/asm/package.json

```diff
@@ -11,6 +11,11 @@
   "homepage": "https://github.com/SAP/spartacus",
   "repository": "https://github.com/SAP/spartacus/tree/develop/feature-libs/asm",
   "license": "Apache-2.0",
+  "exports": {
+    ".": {
+      "sass": "./_index.scss"
+    }
+  },
```

Comment:
with possible backport for 4.3.x, this fragment should not be added (breaks build)